controller/eks-cluster-config-handler.go

```diff
@@ -1087,6 +1101,7 @@ func (h *Handler) updateUpstreamClusterState(upstreamSpec *eksv1.EKSClusterConfi
 	}
 
 	var attemptUpdatingNodegroups bool
```

Comment:
I think it may be confusing that there are two separate chunks of updating node groups logic, but I believe one is creating/deleting node groups and the other is updating node group properties. Can we name `attemptUpdatingNodegroups` to something `updateNodegroupProperties` so it's clear what the two are for? If I'm misunderstanding please clarify.

Reply:
Done